docs/guides/01-getting-started.md

```diff
@@ -103,9 +103,9 @@ layer.addTo(map);
 ```
 
 ### Giving styles to the map
-Using the viz object you can create complex visualizations of your data.
+Using the viz object you can decide how to visualize your data.
 
-The following Viz changes the color and size of the points on our map [See example](http://carto.com/developers/carto-vl/examples/maps/guides/getting-started/basicStyling.html).
+The following Viz changes the color and size of the points on our map. [See example](http://carto.com/developers/carto-vl/examples/maps/guides/getting-started/basicStyling.html).
```

Comment:
And you're still using three different ways when talking about the viz:

1. `viz`
2. **viz object**
3. **Viz**

Pick one and stick to it!